PowerPointLabs/PowerPointLabs/SyncLab/View/SyncFormatPaneItem.cs

```diff
@@ -77,12 +77,14 @@ private void RadioBoxChecked(object sender, RoutedEventArgs e)
         private void OnMouseDoubleClick(object sender, MouseButtonEventArgs e)
         {
             SyncFormatDialog dialog = new SyncFormatDialog(shape, this.formats);
+            dialog.ObjectName = this.Text;
             bool? result = dialog.ShowDialog();
             if (!result.HasValue || !(bool)result)
             {
                 return;
             }
             this.formats = dialog.Formats;
+            this.Text = dialog.ObjectName;
```

Comment:
Suggestion: How about using the default name if the user leaves the dialog field blank?